mslib/msui/mss_pyui.py

```diff
@@ -235,6 +235,12 @@ def __init__(self, mscolab_data_dir=None, *args):
         self.active_flight_track = None
         self.last_save_directory = config_loader(dataset="data_dir")
 
+        # bind keyboard shortcuts to view actions
+        self.actionTopView.setShortcut(QtGui.QKeySequence("Ctrl+h"))
+        self.actionSideView.setShortcut(QtGui.QKeySequence("Ctrl+v"))
+        self.actionTableView.setShortcut(QtGui.QKeySequence("Ctrl+t"))
+        self.actionLinearView.setShortcut(QtGui.QKeySequence("Ctrl+l"))
```

Comment:
can't this be implemented using the QtCreator and the UI file?

Reply:
When set as Ctrl+... in a mac, the shortcuts translate to "meta" which maps to no keys in linux systems. And also, Ctrl+h translates to command+h in mac, which is a system wide shortcut, so setting it here resolves these problems.

Reply:
Or we could find a different shortcut for opening topview and set it through QtCreator itself in a linux/windows system.

Reply:
We should keep your solution, please add a comment string above. e.g. ctrl+h is a special command on Mac, we can't use it in the QtCreator.

Reply:
okay!